src/wasm-features.h

```diff
@@ -36,6 +36,7 @@ struct FeatureSet {
     TailCall = 1 << 7,
     ReferenceTypes = 1 << 8,
     Multivalue = 1 << 9,
+    Default = MVP | MutableGlobals,
```

Comment:
Can you put `Default` after `All` to keep the sequential numbering visually clear?